graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/SchemaGenerator.kt

```diff
@@ -94,10 +91,15 @@ open class SchemaGenerator(internal val config: SchemaGeneratorConfig) {
      * This is helpful for things like federation or combining external schemas
      */
     protected fun addAdditionalTypesWithAnnotation(annotation: KClass<*>) {
-        classScanner.getClassesWithAnnotation(annotation)
-            .map { generateGraphQLType(this, it.createType()) }
-            .forEach {
-                additionalTypes.add(it)
-            }
+        classScanner.getClassesWithAnnotation(annotation).forEach {
+            additionalTypes.add(it.createType())
+        }
     }
+
+    /**
+     * Gererate the GraphQL type for all the additional types. They are generated as non-inputs and not as IDs.
```

Comment:
```suggestion
     * Generate the GraphQL type for all the additional types. They are generated as non-inputs and not as IDs.
```

Reply:
Removed java doc as it is internal for now